content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,16 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+##Java Development Kits Used
+
+Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications. This includes:
+* Container Images
+* Controller Builds
+* Testing Infrastructure
+
+Some of the reasons of choosing Temurin are:
+1. Its availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.
+2. Regular maintainence and long term support provided by the Eclipse Foundation
+3. Additional performance optimizations designed to improve the performance and scalability of the JDK in enterprise-level applications.
```

Comment:
```suggestion
== Java Development Kits Used

Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications.
This includes:
* Container Images
* Controller Builds
* Testing Infrastructure

Some of the reasons for choosing Temurin are:
. Its availability over a wide range of platforms, including different operating systems and architectures, over many different Java SE versions.
. Regular maintenance and long term support provided by the Eclipse Foundation.
. Additional performance optimizations designed to improve the performance and scalability of the JDK in enterprise-level applications.
```
updated to use ascii formatting (we have moved from markdown to asciidoc formatting) and set the ordered list accordingly.